src/nxdoc/getting-started/tracks/discover-nuxeo-platform-apis/setting-up-your-nuxeo-environment.md

```diff
@@ -296,217 +296,71 @@ next_link: nxdoc/setting-up-your-javascript-environment
+You should contact your Nuxeo Administrator or Nuxeo sales representative to get access to this image.
 {{/callout}}
 
-## Homebrew
-
-OS: Mac OS
-
-### Installation
-
-1.  Install Homebrew: Follow [their documentation](http://brew.sh/).
-2.  Install the Nuxeo Platform:
-
-    ```bash
-    $ brew install nuxeo
-    ```
-
-3.  Register your Nuxeo instance on Nuxeo Online Services:
-
-    ```
-    # You don't have a Nuxeo Online Service account
-    $ nuxeoctl register-trial
-    # You already have a Nuxeo Online Service account
-    $ nuxeoctl register
-    ```
-
-    And follow the instructions displayed.
-
-### Setting up Environment
-
-You now have a Nuxeo Platform instance ready to run. At this step, you need to install some addons, in particular the addon Getting started with the Nuxeo Platform.
-
-The addon Getting started with the Nuxeo Platform will add some business logic and documents to your Nuxeo Platform instance, that we will leverage all through this tutorial.
-
-To install a Nuxeo Package:
-
-1.  Get the list of local Nuxeo addons.
-
-    ```
-    $ nuxeoctl mp-list
-    ```
-
-    You get the list of local addons, whose status is downloaded.
-
-2.  Install the addons required for this tutorial.
+1.  Log in docker-private.packages.nuxeo.com
 
     ```
-    $ nuxeoctl mp-install nuxeo-dam nuxeo-web-ui nuxeo-platform-getting-started
+    $ docker login docker-private.packages.nuxeo.com -u <username> -p <token_pass_code>
     ```
 
-3.  Validate the dependency resolution step.
+1. Install the addons required for this tutorial and start Nuxeo in development mode
 
-    ```
-    Dependency resolution:
-      Installation order (3):        nuxeo-dam-6.4.3/nuxeo-web-ui-2.4.0/nuxeo-platform-getting-started-1.3.3
-      Packages to download (3):      nuxeo-web-ui:2.4.0, nuxeo-dam:6.4.3, nuxeo-platform-getting-started:1.3.3
-
-    Do you want to continue (yes/no)? [yes]
-    ```
-
-4.  Check the Nuxeo addons were correctly installed.
-
-    ```
-    $ nuxeoctl mp-list
-    [...]
-    Local packages:
-     addon     started	nuxeo-dam (id: nuxeo-dam-6.4.3)
-     addon     started	nuxeo-web-ui (id: nuxeo-web-ui-2.4.0)
-     addon     started	nuxeo-platform-getting-started (id: nuxeo-platform-getting-started-1.3.3)
-    ```
-
-    The installed addons now have the status started.
-
-5.  Start the server
-
-    ```
-    $ nuxeoctl console
-    ```
-
-    Your server is now running. You benefit from the following features brought by the addon Getting started with the Nuxeo Platform.
-
-The addon brings the following document types:
-{{{multiexcerpt 'studio_configuration_desc' page='Setting up Your Nuxeo Environment'}}}
+{{#> callout type='warning' heading='Dev Mode'}}
+Running Nuxeo in DEV MODE is insecure and not production-ready, and should only be used for local development purposes
+{{/callout}}
 
-{{#> callout type='info' heading='Learn more'}}
+  ```
+  $ docker run --name nuxeo \
+    -p 8080:8080 --rm -v /tmp:/tmp \
```

Comment:
Is there a need to mount the `/tmp` directory from the container to the local `/tmp` directory?

Reply:
No, I'll remove it